layouts/index.html

```diff
@@ -11,13 +11,13 @@ <h1 class="font-sans font-body font-normal">
                 {{ .Params.subhead | markdownify }}
             </p>
             <div class="mt-6">
-                <a id="home-hero-cta" href="{{ relref . "/docs/get-started" }}" class="btn bg-purple-300 hover:bg-purple-200 border-none px-8 py-4 transition-all text-sm rounded-full uppercase shadow-lg" title="Create your first project with Pulumi" track-click>
+                <a href="{{ relref . "/docs/get-started" }}" class="btn bg-purple-300 hover:bg-purple-200 border-none px-8 py-4 transition-all text-sm rounded-full uppercase shadow-lg" title="Create your first project with Pulumi">
```

Comment:
Why not track these?

Reply:
They are tracked, but they are auto-generated ids from the current URL path, the target URL, and what number link they are on the page. So, for example, this link gets an id of `home-docs-get-started-12`